wagtail/project_template/project_name/settings/base.py

```diff
@@ -148,12 +148,8 @@
     "default": {
         "BACKEND": "django.core.files.storage.FileSystemStorage",
     },
-    # ManifestStaticFilesStorage is recommended in production, to prevent
```

Comment:
The comment above the setting saying "with the staticfiles storage updated" should also be removed, as this is no longer true.